src/plugins/ui_actions/README.asciidoc

```diff
@@ -1,14 +1,68 @@
 [[uiactions-plugin]]
 == UI Actions
 
-An API for:
-
-- creating custom functionality (`actions`)
-- creating custom user interaction events (`triggers`)
-- attaching and detaching `actions` to `triggers`.
-- emitting `trigger` events
-- executing `actions` attached to a given `trigger`.
-- exposing a context menu for the user to choose the appropriate action when there are multiple actions attached to a single trigger.
+UI Actions plugins provides API to manage *triggers* and *actions*.
+
+*Triggers* are like events that can be "triggered". For, example one such
+trigger is when somebody applies filters on dashboard.
```

Comment:
applying filter is an action already. example trigger would be somebody clicking something.

`Trigger` is an abstract description of users intent to perform an action (like user clicking on a value inside chart).  It allows us to do runtime binding between code from different plugins.